src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -569,6 +570,47 @@ trait WorkflowComponent {
       deleteWorkflowFailureAttributes(workflowFailureId) andThen
         findInputResolutionsByFailureId(workflowFailureId).delete
     }
+
+
+    object WorkflowStatisticsQueries extends RawSqlQuery {
+      val driver: JdbcDriver = WorkflowComponent.this.driver
+
+      def countWorkflowsPerUserQuery(startDate: String, endDate: String) = {
+        sql"""select min(count), max(count), avg(count), stddev(count)
+                from (
+                  select count(1) as count from WORKFLOW w
+                    join SUBMISSION s on s.ID=w.SUBMISSION_ID
+                    where s.DATE_SUBMITTED between $startDate and $endDate
+                    group by s.SUBMITTER
+                ) as counts""".as[SummaryStatistics].head
+      }
+
+      def countWorkflowsPerSubmission(startDate: String, endDate: String) = {
+        sql"""select min(count), max(count), avg(count), stddev(count)
+                from (
+                  select count(1) as count from WORKFLOW w
+                    join SUBMISSION s on s.ID=w.SUBMISSION_ID
+                    where s.DATE_SUBMITTED between $startDate and $endDate
+                    group by w.SUBMISSION_ID
+                ) as counts""".as[SummaryStatistics].head
+      }
+
+      def workflowRunTimeQuery(startDate: String, endDate: String) = {
+        sql"""select min(seconds), max(seconds), avg(seconds), stddev(seconds)
+                from (
+                  select TIMESTAMPDIFF(SECOND, MIN(a.timestamp), MAX(a.timestamp)) as seconds
+                    from WORKFLOW w join AUDIT_WORKFLOW_STATUS a on w.ID=a.workflow_id
+                    where w.STATUS in ("Succeeded") and a.STATUS in ("Queued","Succeeded")
```

Comment:
assuming our DB triggers stay intact, having a queued and succeeded row is guaranteed. the queued status should always be guaranteed because that's entered when the workflow is first created, and the succeeded status will always be guaranteed because we're only looking at workflows that have the succeeded status to begin with

if you feel strongly about excluding zeroes just in case, i'll add it but i don't think it'll be an actual problem